lib/yabeda/histogram.rb

```diff
@@ -7,7 +7,7 @@ class Histogram < Metric
     option :buckets
 
     def measure(tags, value)
-      all_tags = ::Yabeda::Tags.build(tags)
+      all_tags = ::Yabeda::Tags.build(tags, group)
```

Comment:
Would it be worth extracting this to `Metric` since all three metrics have this? I don't have a good name for it since we already have `tags` for just the keys.